### ISSUE TITLE
GDM Logo low contrast

### ISSUE THREAD

**Issue by Aman9das** [ublue-os/bluefin#1138]
### Describe the bug

The login screen logo doesn't look good at all over the dark gdm theme

### What did you expect to happen?

It should have better contrast for accessibility and better first impressions

### Output of `rpm-ostree status`

```shell
State: idle
AutomaticUpdates: stage; rpm-ostreed-automatic.timer: last run 31min ago
Deployments:
  ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:latest
                   Digest: sha256:f856b59bab5b0f1e35651cd709262b43728b4894f7967cd540f447f06aa917d5
                  Version: 39.20240414.0 (2024-04-14T21:59:19Z)
                     Diff: 7 upgraded, 5 removed, 14 added

● ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:latest
                   Digest: sha256:f848846104584b3c1143f174c3d5e158549c62139dc172414ab76420b981115e
                  Version: 39.20240413.0 (2024-04-13T23:31:20Z)

  ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:latest
                   Digest: sha256:4703c1c6019cd734c09f01ade8918c628f8f4ae27dc3bbc25d5426e53c99c797
                  Version: 39.20240412.0 (2024-04-12T17:04:47Z)
```

### Extra information or context

Kindly add a translucent white background behind the logo

**Comment by Aman9das** [comment]
Thanks
